### PR TITLE
support send keys to active_element

### PR DESCRIPTION
**Pull request by Chris8711**
add a mixin class ActiveElement
support function get_active_element  and send_keys_to_active_element

### CODE REVIEW

File: appium/webdriver/extensions/active_element.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        It usually used after tapping a point and active the element.
+        And use the element.
+
+        Returns:
+
+            `WebElement`: web element
+        """
+        try:
+            return self.execute(W3C_Command.W3C_GET_ACTIVE_ELEMENT)['value']
+        except NoSuchElementException:
```
Comment (mykola-mokhnach) [inline appium/webdriver/extensions/active_element.py:37]: it is ok to throw the exception from this method: https://www.w3.org/TR/webdriver/#dfn-get-active-element

File: appium/webdriver/extensions/active_element.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+            return None
+
+    def send_keys_to_active_element(self, value: str):
```
Comment (mykola-mokhnach) [inline appium/webdriver/extensions/active_element.py:40]: I don't see any actual point in this helper method. also, it is not a part of the spec

Comment (Chris8711) [reply appium/webdriver/extensions/active_element.py:37]: the driver.tap([positions]) will actually  click some element

Comment (Chris8711) [reply appium/webdriver/extensions/active_element.py:37]: I use other tool the get the position on the screenshot.
and use appium client to tap or click the screen.
if some element was tapped. Then I can use get_active_element to do other things

**Comment by mykola-mokhnach** [comment]
I can observe that selenium python lib already has a wrapper for active element:

```
class SwitchTo:
    def __init__(self, driver):
        import weakref
        self._driver = weakref.proxy(driver)

    @property
    def active_element(self) -> WebElement:
        """
        Returns the element with focus, or BODY if nothing has focus.

        :Usage:
            ::

                element = driver.switch_to.active_element
        """
        return self._driver.execute(Command.W3C_GET_ACTIVE_ELEMENT)['value']
```

Wouldn't this be a duplicate then?

**Comment by Chris8711** [comment]
> 



> I can observe that selenium python lib already has a wrapper for active element:
> 
> ```
> class SwitchTo:
>     def __init__(self, driver):
>         import weakref
>         self._driver = weakref.proxy(driver)
> 
>     @property
>     def active_element(self) -> WebElement:
>         """
>         Returns the element with focus, or BODY if nothing has focus.
> 
>         :Usage:
>             ::
> 
>                 element = driver.switch_to.active_element
>         """
>         return self._driver.execute(Command.W3C_GET_ACTIVE_ELEMENT)['value']
> ```
> 
> Wouldn't this be a duplicate then?

yes. I haven't noticed that.
Thank you

**Comment by Chris8711** [comment]
this is duplicated

**Comment by KazuCocoa** [comment]
Btw, sending keys to an active element is now W3C's send_keys action.
https://github.com/appium/appium/issues/16419#issuecomment-1031764419

Get an active element -> calling send_keys to the element is also the same, but the w3c action's one is old send_keys_active_element (`[:post, 'session/:session_id/keys']`) endpoint.

**Pull request closed without merging**